site/README.md

```diff
@@ -23,11 +51,17 @@ cd site
 make server
 ```
 
+The html files will be generated (or updated) into `./public` and a server will be started running on `localhost:4000`. Point your browser there to see the site. Changes to files in `./src` or any `.md` file will trigger a live reload on the site (no need to hit refresh).
+
+> To see what is going on under the hood look at the `Makefile` and at the `./scripts/serve` script. If you are not familiar with browser-sync check out [their website](https://www.browsersync.io/) for details.
+
 ## Publish
 
-From the project root:
+Publishing to `http://code.viget.com/microcosm` can only be done by core contributors with merging privileges. If this is you then publishing is straightforward, from the project root:
```

Comment:
Are there other reasons to use the publish script? Contributors shouldn't submit PR's from the `gh-pages` branch right? We will generate the site ourselves when it's ready.